spec/lib/assigner_spec.rb

```diff
@@ -264,6 +264,21 @@ def assigned_to?(assignee)
         expect(second_assign[:success]).to eq(true)
       end
 
+      it "assignments limit doesn't prevent from reassigning a post" do
```

Comment:
How about?

```suggestion
      it "reassigns even when at the assignments limit" do
```

Reply:
Sounds good, with one detail that we should explicitly mention that this test is for reassigning _a post_. We also reassign _topics_ sometimes.